service/src/main/resources/application.yml

```diff
@@ -8,4 +8,9 @@ management:
       path-mapping.health: healthcheck
       exposure:
         include: info, health, metrics, mappings
-
+storage:
+  platform: ${STORAGE_SOLUTION:LocalMock}
```

Comment:
storage.platform, $STORAGE_SOLUTION

names are inconsistent.

storage.type and $GP2GP_STORAGE_TYPE?